picturepoint/src/AddPhoto/AddPhoto.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react";
+import axios from "axios";
```

Comment:
I rebased to kill the conflict but I don't know what just happened. Good news though, the conflict is gone and src/AddPhoto/AddPhoto.jsx was updated

Reply:
actually, nevermind they are still there :(

Reply:
Now it's fixed